var/spack/repos/builtin/packages/py-distributed/package.py

```diff
@@ -43,28 +44,39 @@ class PyDistributed(PythonPackage):
     depends_on("python@3.8:", when="@2022.2.1:", type=("build", "run"))
     depends_on("py-setuptools", type=("build", "run"))
 
+    # In Spack py-dask+distributed depends on py-distributed, not the other way around. Hence, no need for depends_on("py-dask", ...)
     depends_on("py-click@6.6:", type=("build", "run"))
     depends_on("py-cloudpickle@0.2.2:", type=("build", "run"), when="@:2.16.0")
     depends_on("py-cloudpickle@1.3.0:", type=("build", "run"), when="@2.17.0:2.20.0")
     depends_on("py-cloudpickle@1.5.0:", type=("build", "run"), when="@2.21.0:")
     depends_on("py-jinja2", type=("build", "run"), when="@2022.2.1:")
+    depends_on("py-locket@1:", type=("build", "run"), when="@2022.2.1:")
     depends_on("py-contextvars", type=("build", "run"), when="@2020: ^python@:3.6")
     depends_on("py-msgpack", type=("build", "run"), when="@:2.10.0")
     depends_on("py-msgpack@0.6.0:", type=("build", "run"), when="@2.11.0:")
     depends_on("py-packaging@20.0:", type=("build", "run"), when="@2022.2.1:")
     depends_on("py-psutil@5.0:", type=("build", "run"))
     depends_on("py-six", type=("build", "run"), when="@:1")
     depends_on("py-sortedcontainers@:1,2.0.2:", type=("build", "run"))
+    conflicts_with("py-sortedcontainers@2.0.0,2.0.1", when="@2022.10.2:")
```

Comment:
What is this? I've never heard of `conflicts_with`. Also, this is already handled by `depends_on`